tensorflow_addons/custom_ops/image/cc/kernels/adjust_hsv_in_yiq_op.cc

```diff
@@ -168,4 +168,4 @@ REGISTER_KERNEL_BUILDER(
 #endif
 
 }  // end namespace addons
-}  // namespace tensorflow
\ No newline at end of file
+}  // namespace tensorflow
```

Comment:
Could you revert the change of this line? I mean do any change to this file actually :-) Thanks.